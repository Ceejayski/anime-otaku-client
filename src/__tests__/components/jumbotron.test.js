import { create } from 'react-test-renderer';
import Jumbroton from '../../components/jumbotron';

test('renders Correctly', () => {
  const Placeholder = () => (

    <Jumbroton />
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
