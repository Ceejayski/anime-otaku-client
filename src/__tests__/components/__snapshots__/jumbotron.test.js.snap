// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
<div
  className="w-100 jumbotron d-flex align-items-center mb-5"
>
  <div
    className="container"
  >
    <div
      className="text-white"
    >
      <h1>
        Best Anime Review Site
      </h1>
      <p
        className="w-100 lead"
        style={
          Object {
            "maxWidth": "700px",
          }
        }
      >
        "Religion, ideology, resources, land, spite, love or just because… No matter how pathetic the reason, it’s enough to start war. War will never cease to exist… reasons can be thought up after the fact… Human nature pursues strife."
        <strong>
          ~ Yahiko Paine (Naruto Shippuden)
        </strong>
      </p>
      <a
        className="btn btn-lg text-white explore-btn"
        onClick={[Function]}
      >
        Explore
      </a>
    </div>
  </div>
</div>
`;
